src/cpp/session/modules/rmarkdown/SessionRMarkdown.cpp

```diff
@@ -271,6 +271,21 @@ enum RenderTerminateType
 std::vector<std::string> s_renderOutputs(kMaxRenderOutputs);
 int s_currentRenderOutput = 0;
 
+std::string parsableRStudioVersion()
+{
+   std::string version(RSTUDIO_VERSION_MAJOR);
+   version.append(".")
+      .append(RSTUDIO_VERSION_MINOR)
+      .append(".")
+      .append(RSTUDIO_VERSION_PATCH)
+      .append(".")
+      .append(boost::regex_replace(
+         std::string(RSTUDIO_VERSION_SUFFIX),
+         boost::regex("[a-zA-Z\\-+]"),
```

Comment:
What will this do with Pro versions? (it kind of looks like it'll turn a suffix of "+388-pro1" into "3881")

Reply:
It's `+338.pro1`, so it would become `338.1`

Reply:
👍